eng/dogfood.sh

```diff
@@ -12,6 +12,7 @@ scriptroot="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
 InitializeTools
 
+export PATH="$RepoRoot/.dotnet:$PATH"
```

Comment:
>$RepoRoot/.dotnet [](start = 13, length = 17)

use `$DOTNET_INSTALL_DIR`

Reply:
Fixed

Reply:
Now reverted. Unnecessary with change to arcade to set PATH